src/lib/elements/ResponsiveSidebar.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Grid, Sidebar, Button, Segment } from 'semantic-ui-react';
+
+export class ResponsiveSidebar extends React.Component {
+  render() {
+    const closeSidebarBtnRef = React.createRef();
+    const { columnWidth, open, setOpen, children } = this.props;
+
+    return (
+      <>
+        <Grid.Column width={columnWidth} only="mobile tablet">
+          <Sidebar
+            as={Segment}
+            animation="overlay"
+            visible={open}
+            width="wide"
+            onHide={() => setOpen(false)}
+            onShow={() => closeSidebarBtnRef.current.focus()}
+          >
+            <Grid>
+              <Grid.Column align="right">
+                <Button
+                  basic
+                  icon="close"
+                  size="small"
+                  onClick={() => setOpen(false)}
+                  aria-label="Close filter"
+                  ref={closeSidebarBtnRef}
+                />
+              </Grid.Column>
+            </Grid>
+
+            {children}
+          </Sidebar>
+        </Grid.Column>
+
+        <Grid.Column width={columnWidth} only="computer">
+          {children}
+        </Grid.Column>
+      </>
+    );
+  }
+}
+
+ResponsiveSidebar.propTypes = {
+  columnWidth: PropTypes.number,
```

Comment:
```suggestion
  width: PropTypes.number,
```
I think it is better to keep naming consistency with semantic